docs/wallets/cli.mdx

```diff
@@ -0,0 +1,275 @@
+always recommended to explore the help menu to learn more:
+
+```
+$ helium-wallet create basic --help
+
+helium-wallet-create-basic 1.5.6
+Create a new basic wallet
+
+USAGE:
+    helium-wallet create basic [FLAGS] [OPTIONS]
+
+FLAGS:
+        --force      Overwrite an existing file
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+        --key-type <key-type>    The type of key to generate (ecc_compact/ed25519) [default: ed25519]
+        --network <network>      The network to generate the wallet (testnet/mainnet) [default: mainnet]
+    -o, --output <output>        Output file to store the key in [default: wallet.key]
+        --seed <seed>            Use a BIP39 or mobile app seed phrase to generate the wallet keys [possible values:
+                                 bip39, mobile]
+```
+
+We'll go through a few examples next.
+
+### Managing Multiple Wallets
+
+You can specify a more precise location or filename when _creating a wallet_ by using the
+`--output` flag. eg:
+
+`helium-wallet create basic --output ~/my-wallets/wallet-5.key`
+
+When _using a wallet_, you can specify a file other than the default `./wallet.key` by using
+the the global option `--file`. eg:
+
+`helium-wallet -f ~/my-wallets/wallet-5.key pay <AMOUNT> <ADDRESS>`
+
+### Sharded Wallet
+
+Sharding wallet keys is supported via Shamir's Secret Sharing. A key can be broken into N shards such that
+recovering the original key needs K distinct shards. This can be done by passing options to create:
+
+```
+    helium-wallet create sharded -n 5 -k 3
+```
+
+This will create wallet.key.1 through wallet.key.5 (the base name of the wallet file can be supplied with
+the `--output` parameter).
+
+When keys are sharded using verify will require at least K distinct keys.
+
+The `--seed` option described above can also be used to construct a sharded wallet.
+
+#### Implementation details
+
+A ed25519 key is generated via libsodium. The provided password is run through PBKDF2, with a configurable
+number of iterations and a random salt, and the resulting value is used as an AES key. When sharding is
+enabled, an additional AES key is randomly generated and the 2 keys are combined using a sha256 HMAC into
+the final AES key.
+
+The private key is then encrypted with AES256-GCM and stored in the file along with the sharding information,
+the key share (if applicable), the AES initialization vector, the PBKDF2 salt and iteration count and the
+AES-GCM authentication tag.
+
+### Import Seed
+
+The `--seed` flag enables you to import a wallet generated elsewhere. For example, you may want to import
+you key from the mobile wallet, so that you can submit transactions that may not be supported in the mobile
+wallet.
+
+Enter the command:
+
+```
+$ helium-wallet create basic --seed
+```
+
+After you press enter, you will be prompted for your seed words.
+
+::: info
```

Comment:
Guessing prettier did this

Reply:
whats wrong with it?

Reply:
you need a line between this info wrapper I think, it isn't rendering right now